admin/venues.php

```diff
@@ -29,7 +29,12 @@ function gigpress_venues() {
 	<?php
 	
 	$venue_id = (isset($_REQUEST['venue_id'])) ? $wpdb->prepare('%d', $_REQUEST['venue_id']) : '';
-	
+
+	/**
+	 * Filter to allow the venue city field to be set as not required.
+	 */
```

Comment:
```suggestion
	/**
	 * Filter to allow the venue city field to be set as not required.
	 *
 	 * @since TBD
 	 *
 	 * @param boolean If the city should be required.
 	 */
```